### PR TITLE
Remove unused mixed type

### PR DESCRIPTION
**Pull request by beeender**
The mixed type is not used by now and it is very unlikely going to be
used in the future.

### CODE REVIEW

**Comment by cmelchior** [comment]
Some RealmProcessor unit tests seems to fail, but apart from that 👍

File: realm/realm-jni/src/tablebase_tpl.hpp [hunk 1]
```diff
@@ -17,14 +17,15 @@
 #ifndef REALM_JNI_TABLEBASE_TPL_HPP
 #define REALM_JNI_TABLEBASE_TPL_HPP
 
+#include <realm.hpp>
 
 template <class T>
 jbyteArray tbl_GetByteArray(JNIEnv* env, jlong nativeTablePtr, jlong columnIndex, jlong rowIndex)
```
Comment (stk1m1) [inline realm/realm-jni/src/tablebase_tpl.hpp]: is it possible to inline this as this isn't used in many places?

**Comment by stk1m1** [comment]
👍

**Comment by cmelchior** [comment]
👍

**Comment by kneth** [comment]
:+1:

**Pull request merged**